soroban-env-host/src/budget.rs

```diff
@@ -804,6 +809,26 @@ impl Default for BudgetImpl {
                     cpu.const_term = 130;
                     cpu.linear_term = 0;
                 }
+                ContractCostType::ComputeKeccak256Hash => {
+                    cpu.const_term = 3322;
+                    cpu.linear_term = 46;
+                }
+                ContractCostType::ComputeEcdsaSecp256k1Key => {
+                    cpu.const_term = 56525;
+                    cpu.linear_term = 0;
+                }
+                ContractCostType::ComputeEcdsaSecp256k1Sig => {
+                    cpu.const_term = 250;
+                    cpu.linear_term = 0;
+                }
+                ContractCostType::VerifyEcdsaSecp256k1Sig => {
```

Comment:
Wow at how expensive this is. Even more than VM instantiation.

Reply:
It'd be a whole lot slower in wasm!